src/assisted_test_infra/test_infra/utils/utils.py

```diff
@@ -356,21 +357,59 @@ def touch(path):
         os.utime(path, None)
 
 
-def config_etc_hosts(cluster_name: str, base_dns_domain: str, api_vip: str):
+def config_etc_hosts(cluster_name: str, base_dns_domain: str, api_vip: str = None, ingress_vip: str = None):
+    suffix = cluster_name + "." + base_dns_domain
+
+    dns = {}
+
+    if api_vip:
+        dns["api." + suffix] = api_vip
```

Comment:
```suggestion
        dns[f"api.{suffix}"] = api_vip
```
same goes for the others